lib/EpdFont/scripts/fontconvert.py

```
@@ -882,6 +883,155 @@ def cp_label(cp):
+    alt_total_compressed = len(alt_compressed_bitmap_data)
+    print(f"// Alt compression: {alt_total_size} -> {alt_total_compressed} bytes "
+          f"({100*alt_total_compressed/alt_total_size:.1f}%), {len(alt_compressed_groups)} groups", file=sys.stderr)
+
+    # Output alt arrays
+    print(f"static const uint8_t {font_name}AltBitmaps[{len(alt_compressed_bitmap_data)}] = {{")
+    for c in chunks(alt_compressed_bitmap_data, 16):
+        print("    " + " ".join(f"0x{b:02X}," for b in c))
+    print("};\n")
+
+    print(f"static const EpdGlyph {font_name}AltGlyphs[] = {{")
+    for i, g in enumerate(alt_glyph_props):
```

Ruff (0.15.10)

[warning] Loop control variable `i` not used within loop body

(B007)